wurst/_handles/Player.wurst

```diff
@@ -210,6 +210,55 @@ public function player.sendSyncData(string prefix, string data) returns bool
 public function player.showChatMessage(int recipient, string message)
 	BlzDisplayChatMessage(this, recipient, message)
 
+/** Sets the maximum allowed hero amount for a player */
+public function player.setHeroMaxAllowed(int amount)
+	SetPlayerTechMaxAllowed(this, 'HERO', amount)
+		
+/** Change tooltip (Name) of given ability for player locally.*/
+public function player.setAbilityTooltip(int abilityId, string abilityName, int abilityLevel)
```

Comment:
These ability-wide changes are probably more appropriate as methods of a new tuple type, `ability(int id)`

Reply:
I kinda cannot see any added benefit here.
Can you elaborate more?
Also there can't be tuple "ability" cause there is new native "ability"

Reply:
> I kinda cannot see any added benefit here.
> Can you elaborate more?

Type safety

namespace cleanliness (don't add stuff to `player`)

> Also there can't be tuple "ability" cause there is new native "ability"

Ok, maybe as extension-functions for `ability` then